sdk/compute/Azure.ResourceManager.Compute/src/Customize/VirtualMachineScaleSetResource.cs

```diff
@@ -55,5 +55,13 @@ public virtual ArmOperation Deallocate(WaitUntil waitUntil, VirtualMachineScaleS
                 throw;
             }
         }
```

Comment:
same comment should apply to the above methods too, could we change them as well?